bedrock/base/templates/includes/global-nav.html

```diff
@@ -134,6 +137,24 @@ <h3 class="summary" data-id="firefox">
             </div>
           </div>
         </li>
+        <li class="item-pocket">
+          <h3 class="summary" data-id="pocket">
+            <a href="?utm_source=www.mozilla.org&amp;utm_medium=referral&amp;utm_campaign=global-nav&amp;utm_content=pocket" data-link-type="nav" data-link-position="side" data-link-name="expand" data-link-group="Pocket">{{ _('Pocket') }}</a>
```

Comment:
This link is missing the actual base URL. It just reloads the current page with those GET parameters, when it should take you to getpocket.com from mozilla.org.